packages/fast-check/package.json

```diff
@@ -7,12 +7,11 @@
   "exports": {
     "./package.json": "./package.json",
     ".": {
+      "types": "./lib/types/fast-check.d.ts",
```

Comment:
It would result into an invalid definition of the typings for TypeScript. Let's run the CI against it to confirm but I expect the runs of type-check backed by TypeScript with NodeNext to fail and publint to complain.

Reply:
I think there is probably a problem on effect's side regarding it's support for esm/NodeNext.

Reply:
But in theory (and it's not the case for @effect/schema), mjs and cjs files should never share the same d.ts definition. I had a long public chat about that specific issue on Twitter at: https://twitter.com/AndaristRake/status/1695550761600503827?t=EkApRjlL8ZmEYqZtKtCToA&s=19.